src/api/transformers/ContentTransformer.ts

```diff
@@ -176,8 +178,6 @@ const contentData = {
     paymentMethods: ["data", []],
     showAbsorbFee: ["option", "show-absorb-fee", "bool"],
     showNoContribution: ["data", false],
-    stripeCountry: ["readonly", () => config.stripe.country],
-    stripePublicKey: ["readonly", () => config.stripe.publicKey],
```

Comment:
I've just realised that we'll need to keep these here for one release as Fumaca and IRPI are using this endpoint to fetch the `stripePublicKey`. So we'll do one release with `stripePublicKey` in both `join` and `payment`, then I'll ask them to migrate to the new endpoint, then we can remove them from `join`.

Could add a `@deprecated` or something to make it clear?

Reply:
okay

Reply:
done